documentation/features/cpuset.md

```diff
@@ -0,0 +1,55 @@
+# Cpuset cgroup.
+
+From Kernel documentation:
+
+_" Cpusets provide a mechanism for assigning a set of CPUs and Memory Nodes to
+a set of tasks."_
+
+The Kata agent brings compatibility the cgroup cpuset CPU on the guest side.
+
+The cpuset CPU cgroup will be applied on two events:
+
+- containers creation.
+
+- container update.
+
+When the runtime request to apply cpuset cgroup to the agent, the amout of
+vcpus available may be not the same to the required in the request.
+
+This is becasue the request from the agent client (the Kata runtime) pass
+cpusets that were requested to be placed on the host. This is to isolate the
+container workload on some specific host CPUs. The runtime pass the
+requested cpuset to the agent, that will try to apply the cgroup cpuset on the
+guest.
+
+Because the runtime will only hot-plug and amount the CPUs calculated based in
```

Comment:
Lines 25-27 suggested rewrite (not sure if you want "host" or "guest" at the end of the sentence): 
The runtime only hot-plugs and calculates the CPUSs based on the container period and quota. This is why the VM will not have the same amount of CPUs as the host.